source/lib-intro.tex

```diff
@@ -1119,7 +1119,7 @@
 programs having only a single thread of execution.
 
 \pnum
-Within this subclause, \ref{organization} describes the library's contents and
+\ref{organization} describes the library's contents and
```

Comment:
Is this a case for "Subclause" at the start of the sentence? Maybe repeat "subclause" for the other parts of the enumeration.

Reply:
I decided to leave off the "Subclause" here for consistency with what we already do two paragraphs later. Happy to change it as suggested if you would prefer.

Reply:
Fine, no big deal either way.